distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -173,3 +173,8 @@ karaf.secured.services=(&(osgi.command.scope=*)(osgi.command.function=*))
 #org.osgi.framework.trust.repositories=${karaf.home}/etc/trustStore.ks
 ws-security.subject.cert.constraints = .*
 security.audit.roles=group,admin,manager,viewer,system-admin,system-history,systembundles
+
+#
+# Disable Hazelcast version check
+#
+hazelcast.version.check.enabled=false
```

Comment:
Looks like the documentation still includes that option.  Maybe just that constant in that class has been deprecated.

http://docs.hazelcast.org/docs/3.5/manual/html/systemproperties.html